pyproject.toml

```diff
@@ -36,7 +36,7 @@ dependencies = [
   "plopp",
   "pythreejs",
   "orsopy",
-  "sciline>=23.9.1",
+  "sciline @ git+https://github.com/scipp/sciline@main",
```

Comment:
I think the `sciline` release has been made now, so we should go back to not using `main` and re-run the `tox -e deps`?